ext_localconf.php

```diff
@@ -27,6 +27,8 @@
 
 if (TYPO3_MODE === 'BE') {
     $GLOBALS ['TYPO3_CONF_VARS']['SC_OPTIONS']['t3lib/class.t3lib_tcemain.php']['processDatamapClass']['ats'] = \PAGEmachine\Ats\Hook\DataHandlerJobGroups::class;
+
+    $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['extbase']['commandControllers'][] = \PAGEmachine\Ats\Command\ApplicationsCommandController::class;
```

Comment:
You can do this outside of the condition, command controllers cannot be used in FE anyways.